internal/btf/btf.go

```diff
@@ -544,6 +544,35 @@ func NewHandle(spec *Spec) (*Handle, error) {
 	return &Handle{fd}, nil
 }
 
+// NewHandleFromID returns the BTF handle for a given id.
+//
+// Returns ErrNotExist, if there is no BTF with the given id.
```

Comment:
Document that this requires `CAP_SYS_ADMIN`: https://elixir.bootlin.com/linux/v5.13.2/source/kernel/bpf/syscall.c#L3846

Reply:
Addressed in 51f3b3a